inventory/environ.py

```diff
@@ -329,10 +330,29 @@ def getSplunkApps(vars_scope):
     vars_scope["splunk"]["apps_location"] = list(appSet)
 
 def getSecrets(vars_scope):
+    """
+    Parse sensitive passphrases
+    """
     vars_scope["splunk"]["password"] = os.environ.get('SPLUNK_PASSWORD', vars_scope["splunk"]["password"])
     vars_scope["splunk"]["pass4SymmKey"] = os.environ.get('SPLUNK_PASS4SYMMKEY', vars_scope["splunk"].get("pass4SymmKey"))
     vars_scope["splunk"]["secret"] = os.environ.get('SPLUNK_SECRET', vars_scope["splunk"].get("secret"))
 
+def getLaunchConf(vars_scope):
+    """
+    Parse key/value pairs to set in splunk-launch.conf
+    """
+    launch = {}
+    if not "launch" in vars_scope["splunk"]:
+        vars_scope["splunk"]["launch"] = {}
+    # From default.yml
+    if type(vars_scope["splunk"]["launch"]) == dict:
+        launch.update(vars_scope["splunk"]["launch"])
+    # From environment variables
```

Comment:
I want to definitely add documentation about how to use this.  Can you make a separate page
"Creating a custom splunk_launch.conf" or something like that - then fill out with the usage and an example?

Reply:
Can do